src/main/webapp/services/CommentService.js

```diff
@@ -0,0 +1,112 @@
+import axios from 'axios';
+import { getIntegrationBaseUrl } from 'symphony-integration-commons';
+import BaseService from './BaseService';
+
+const unexpectedErrorDialog = require('../templates/unexpectedErrorDialog.hbs');
+const forbiddenDialog = require('../templates/forbiddenDialog.hbs');
+const notFoundDialog = require('../templates/issueNotFoundDialog.hbs');
+const errorDialog = require('../templates/errorDialog.hbs');
+const commentDialog = require('../templates/commentDialog.hbs');
+
+export default class CommentService extends BaseService {
+  constructor(serviceName) {
+    super(serviceName);
+    this.baseUrl = getIntegrationBaseUrl();
+    this.comment = '';
+  }
+
+  openCommentDialog(data, serviceName) {
+    const dialogsService = SYMPHONY.services.subscribe('dialogs');
```

Comment:
Can `this.dialogsService` be declared and instantiated in the BaseService?
Which has least cost?